test/razorpay/test_payment.rb

```diff
@@ -23,6 +26,67 @@ def test_payments_should_be_available
       assert_equal 'card', payment.method, 'Payment method is accessible'
     end
 
+    def test_payments_fetch_downtime
+      stub_get( %r{payments/downtimes$}, 'downtimes_collection')
+      payments = Razorpay::Payment.fetchPaymentDowntime
+      assert_instance_of Razorpay::Collection, payments, 'Payment not an instance of Razorpay::Payment class'
+      assert !payments.items.empty?, 'downtimes should be more than one'
```

Comment:
refute_empty

Reply:
Also pls confirm from someone if downtime can return empty response

Reply:
yes if its return empty then it would be something like this 

`{
      "entity": "collection",
      "count": 0,
      "items": [ ]
    }`